src/font/lv_font_fmt_txt.h

```diff
@@ -184,6 +184,8 @@ typedef struct {
      * from `lv_font_fmt_txt_bitmap_format_t`
      */
     uint16_t bitmap_format  : 2;
+
+    uint8_t stride;
```

Comment:
With bpp=8 the character width would be limited to 255. 
```suggestion
    uint16_t stride;
```